frontend/src/views/NotificacaoForm.vue

```diff
@@ -5,6 +5,18 @@
     </h3>
     <base-page>
       <v-form ref="form">
+        <identificacao-notificante
+          :notificacao="notificacao"
+          @update:unidadeSaudeId="updateUnidadeSaude('unidadeSaudeId', $event)"
+          @update:nomeNotificador="updateNomeNotificador($event)"
+          @update:profissaoId="updateProfissaoId('profissaoId', $event)"
```

Comment:
Isso aqui também vai setar sempre um valor fixo, altere pra
```suggestion
          @update:profissaoId="updateProfissaoId"
```